src/middleware/analytics.ts

```diff
@@ -1,7 +1,15 @@
 import axios from 'axios';
+import { NextFunction, Request } from 'express';
 import { GA_TRACKING_ID, GA_API_SECRET, GA_URL, DEBUG_GA_URL, isProduction as isProductionConfig } from '../config';
 
-const trackEvent = ({ clientIdentifier, category, action, keyword }) =>
+interface Props {
```

Comment:
`Props` is more for React components
```suggestion
interface TrackingEvent {
```